docs/Scaling/Docker Swarm/docker-swarm.md

```diff
@@ -0,0 +1,72 @@
+--8<-- "References/abbreviations.md"
+
+### Introduction
+
+!!! important
+Not for production use!!
+
+This guide helps to quickly deploy and test the Open AMT Cloud Toolkit scalability. For this deployment, docker swarm is used and assumes all the images are built and tested with docker compose, if not please refer to [**Docker Compose**](../../Docker/dockerLocal.md). Also, if the images are not pushed to any registory, it cannot be deployed on other systems. 
+
+### Deploy the stack to the swarm
+
+!!! important "Important - For Linux"
+
+Prefix with sudo to docker commands, if the docker group is not added. Please refer to [**Add docker group**](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
```

Comment:
```suggestion
If running the following commands on Linux, we assume the user is added to the docker group. For instructions on how to do this please refer to [**Add docker group**](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user). Otherwise, you can prefix each command with `sudo`.
```